src/docs/product/issues/index.mdx

```diff
@@ -32,6 +32,26 @@ From the **Issues** page, you can begin to triage. The page is organized into ta
 
 Learn more about triaging issues and their different states in [Issue States and Triage](/product/issues/states-triage/).
 
+### Display Options
+
```

Comment:
I understand why you placed this where you did, but it interrupts the flow a bit and since most of this content is about the Events as % option, I think it's safe to put it right at the top. Generally we want to put this note near the heading anyway.

```suggestion

<Note>

  The "Events as %" display option is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

  If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

```